src/runtime/basic-runtime-test.js

```diff
@@ -976,5 +976,16 @@ describes.realWin('BasicConfiguredRuntime', {}, (env) => {
       expect(isExperimentOn(win, ExperimentFlags.LOGGING_AUDIENCE_ACTIVITY)).to
         .be.false;
     });
+
+    it('should enable Google controlled metering on configuration', () => {
```

Comment:
I probably would phrase "Google controlled" as "METERED_ BY_GOOGLE" to be more specific, here and in the symbols/constants. It's better to avoid introducing a new phrase/concept like "Google controlled" in favor of an existing one, wherever possible.

Reply:
Fair point. renamed to `enableMeteredByGoogle`